src/main/kotlin/com/group/libraryapp/service/user/UserService.kt

```diff
@@ -24,6 +24,12 @@ class UserService(
     userRepository.save(newUser)
   }
 
+  @Transactional
+  fun saveUser_123(request: UserCreateRequest) {
```

Comment:
⚠️ **[detekt]** <detekt.FunctionNaming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function names should match the pattern: [a-z][a-zA-Z0-9]*